### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by CongSon**

### CODE REVIEW

**Comment by ChiThanh93** [comment]
ok

**Comment by ChiThanh93** [comment]
file gitignore

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Fels214</title>
+    <%= csrf_meta_tags %>
+
+    <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track": "reload" %>
```
Comment (nhunght) [inline app/views/layouts/application.html.erb]: media: :all

File: test/test_helper.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../../config/environment', __FILE__)
+require 'rails/test_help'
```
Comment (nhunght) [inline test/test_helper.rb]: double quote

**Pull request merged**